test/cache-interceptor/cache-stores.js

```diff
@@ -213,46 +213,6 @@ function cacheStoreTests (CacheStore) {
   })
 }
 
-test('MemoryCacheStore locks values properly', async () => {
-  const store = new MemoryCacheStore()
-
-  const request = {
-    origin: 'localhost',
-    path: '/',
-    method: 'GET',
-    headers: {}
-  }
-
-  const requestValue = {
-    statusCode: 200,
-    statusMessage: '',
-    rawHeaders: [Buffer.from('1'), Buffer.from('2'), Buffer.from('3')],
-    cachedAt: Date.now(),
-    staleAt: Date.now() + 10000,
-    deleteAt: Date.now() + 20000
-  }
-
-  const writable = store.createWriteStream(request, requestValue)
-  notEqual(writable, undefined)
-
-  // Value should now be locked, we shouldn't be able to create a readable or
-  //  another writable to it until the first one finishes
-  equal(store.get(request), undefined)
-  equal(store.createWriteStream(request, requestValue), undefined)
-
-  // Close the writable, this should unlock it
-  writeResponse(writable, ['asd'])
-
-  // Stream is now closed, let's lock any new write streams
-  const result = store.get(request)
-  notEqual(result, undefined)
-
-  // Consume & close the result, this should lift the write lock
-  await readResponse(result)
-
-  notEqual(store.createWriteStream(request, requestValue), undefined)
-})
```

Comment:
Why this test was removed? Is this functionality not needed anymore?

Reply:
IMHO. We don't need locking. The feature is removed.